staramr/results/AMRDetectionSummary.py

```diff
@@ -49,12 +53,41 @@ def _include_negatives(self, df):
         negative_names_set = names_set - result_names_set
         negative_entries = pd.DataFrame([[x, 'None'] for x in negative_names_set],
                                         columns=('Isolate ID', 'Gene')).set_index('Isolate ID')
+        
+        return df.append(negative_entries, sort=True)
+
+    def _include_detailed_negatives(self, df, ds):
```

Comment:
Could you name the parameters something more descriptive than `df` and `ds` (e.g,. `resistance_frame` and `plasmid_frame`)?

Reply:
Yeah, I renamed all variables that has something like `df`, `ds`, or `dt` in commit 1110972

Reply:
Thanks so much :smile: